custom_components/thermal_comfort/config_flow.py

```diff
@@ -4,9 +4,15 @@
 import logging
 
 from homeassistant import config_entries
-from homeassistant.const import CONF_NAME
+from homeassistant.const import (
+    CONF_NAME,
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
```

Comment:
we should use SensorDeviceClass

Reply:
I'll check more careful device classes.

Current version works well for my installation, but looks like not for any.
Inside `async_get_device_class_lookup` it do
```python
            device_class = entity.device_class or entity.original_device_class
            domain_device_class = (entity.domain, device_class)
```
so it looks like natural device class should be used.

Will review device filtering.

Reply:
[# DEVICE_CLASS_* below are deprecated as of 2021.12](https://github.com/home-assistant/core/blob/e92078cf507089908d103b358acf12230ec4312d/homeassistant/const.py#L274)

Reply:
https://github.com/home-assistant/core/blob/e92078cf507089908d103b358acf12230ec4312d/homeassistant/const.py#L274

Reply:
Oh... Missed deprecation info...